src/main/java/edu/ucsb/cs156/happiercows/entities/Commons.java

```diff
@@ -32,6 +36,7 @@ public class Commons {
     private double degradationRate;
     private int carryingCapacity;
     private int capacityPerUser;
+    private int effectiveCapacity;
```

Comment:
adding effectiveCapacity to the commons creates a data redundancy since this field is derived from capacityPerUser, and from the number of users in the userCommons table.

This is just as bad as adding numUsers or numCows to the Commons table.  It introduces a trap for future developers. It's bad coding practice, and cannot be merged.

If you want the points for this feature, please reimplement it in the way that I outlined in Slack channel.